policy/modules/services/container.if

```diff
@@ -1070,6 +1256,24 @@ interface(`container_use_container_ptys',`
 	allow $1 container_devpts_t:chr_file rw_term_perms;
 ')
 
+########################################
+## <summary>
+##	Watch container ptys.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`container_watch_container_ptys',`
+	gen_require(`
+		type container_devpts_t;
+	')
+
+	allow $1 container_devpts_t:chr_file { watch };
```

Comment:
No braces for a single perm (i.e. `watch` not `{ watch }`).

Reply:
fixed